### PR TITLE
safehax: yeet it

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**
- universal-otherapp was fixed on 9.3-9.5, rendering these pages completely useless.

### CODE REVIEW

File: _data/navigation/en_US.yml [hunk 1]
```diff
@@ -79,9 +79,6 @@ sidebar_pages:
   -
     title: Flashing ntrboot (NDS)
     url: flashing-ntrboot-(nds)
-  -
-    title: Homebrew Launcher (PicHaxx)
-    url: homebrew-launcher-(pichaxx)
```
Comment (lilyuwuu) [inline _data/navigation/en_US.yml:84]: me when i remove a page we're actively using

**Comment by lifehackerhansol** [comment]
This commit name leaves a lot to be desired... technically universal otherapp uses safehax as well. Oh well.

File: _data/navigation/en_US.yml (hunk 1 above)
Comment (lifehackerhansol) [reply _data/navigation/en_US.yml:84]: I can't read.

No better than some of our users...

**Review by lilyuwuu — APPROVED** [review]
lgtm

**Pull request merged**